src/lib.rs

```diff
@@ -568,8 +568,8 @@ mod tests {
 
         let resp = agent.get("http://www.google.com/").call().unwrap();
         assert_eq!(
-            "text/html; charset=ISO-8859-1",
-            resp.header("content-type").unwrap()
+            "text/html;charset=ISO-8859-1",
+            resp.header("content-type").unwrap().replace("; ", ";")
```

Comment:
What's going on here?

Reply:
Response sometimes contains `text/html;charset=ISO-8859-1`, sometimes `text/html; charset=ISO-8859-1` (note the space after the semicolon); making the test flaky.